doc/src/_templates/sidebar/versions.html

```diff
@@ -0,0 +1,56 @@
+<!-- Versions Menu. In order to work, there should be a file called versions.json one level up from the documentation listing the "dev" and "latest" version numbers, like
+{
+    "latest": "1.10",
+    "dev": "1.11.dev"
+}
+-->
+<div class="sidebar-tree sidebar-versions">
+  <h5 class="versions-header">Documentation Version</h4>
```

Comment:
I would suggest adding a padding-left to this, to align it with the rest of the sidebar content. You might want to consider if you want this to look like sidebar captions instead of a heading.

Reply:
I'll have to play with it some more. I'm OK with the way it currently looks but I'll see if it looks better this way. Note that I'm not a designer and my HTML/Javascript/CSS skills are terrible, so if you see something that looks off, it's most likely unintentional so please let me know.

Reply:
Actually this was supposed to have the exact same `padding` as the rest of the sidebar, but my editor inserted an incorrect line break in the CSS and messed it up. Should look better now.